.snippets/code/builders/json-rpc/debug-trace/terminal/start-up-logs.md

```diff
@@ -0,0 +1,25 @@
+<div id="termynal" data-termynal>
+  <span data-ty="input"><span class="file-path"></span>docker run --network host  \
+    <br>-u $(id -u ${USER}):$(id -g ${USER}) \
+        moonbeamfoundation/moonbeam-tracing:v0.36.0-2801-7ae4 \
```

Comment:
I think it is not possible to use variables here right? Just be mindful of the new version

Reply:
No, can't use variables in snippets unfortunately. Haha guess these changes were implemented a while ago 😅 

thanks!